data/src/main/java/com/linkedin/util/CustomTypeUtil.java

```diff
@@ -91,4 +94,18 @@ public static String getJavaCoercerClassFromSchema(final DataSchema schema)
     return (String) o2;
 
   }
+
+  private static Class<?> getBindingClass(String javaCoercerClassName, ClassLoader classLoader, DataSchema schema)
+  {
+    final Class<?> bindingClass;
+    try
+    {
+      bindingClass = Class.forName(javaCoercerClassName, false, classLoader);
```

Comment:
You can simplify this by returning here.

Reply:
Updated